app/models/schedule.rb

```diff
@@ -15,12 +15,15 @@ class Schedule < ActiveRecord::Base
   validate :valid_time, :valid_room
 
   scope :with_room, ->(room, id){where(room: room).where.not id: id}
+  scope :order_start_time, ->{order start_time: :desc}
   query = "(start_time <= :start_time AND finish_time >= :finish_time)
           OR (start_time > :start_time AND start_time < :finish_time)
           OR (finish_time > :start_time AND finish_time < :finish_time)"
   scope :filte_timer, ->(start, finish){where(query, start_time: start, finish_time: finish)}
   scope :today_schedule, ->{where("start_time LIKE ?", "%#{Time.now.to_date.to_s}%")}
   scope :filter_by_room, ->(room_id){where room_id: room_id if room_id.present? && room_id != "all"}
+  scope :my_schedule, ->(user_id, time_now){where(" ? <= start_time  AND user_id = ?", time_now, user_id)}
```

Comment:
You can using:
scope :my_schedule, ->user_id{where(" ? <= start_time  AND user_id = ?", Time.zone.now, user_id)}